GeoLib/PointVec.h

```diff
@@ -143,6 +145,10 @@ class PointVec : public TemplateVec<Point>
 	 */
 	std::vector<std::size_t> _pnt_id_map;
 
+	/// The reverse map to the name to id map, for fast lookup of the name to a
+	/// given point id.
+	std::vector<std::string> _id_to_name_map;
```

Comment:
sorry it was other way around. however i still assume `std::vector<std::set<std::size_t,std::string>>` or `std::map<std::size_t, std::string>` can also speed up (of course not the fastest, but could be enough fast), because comparison of integers is much faster than that of strings (i mean compared to `std::map<std::string, std::size_t>`).